app/src/main/java/kumamotone/sample/eremotethingssample/MainActivity.kt

```diff
@@ -115,24 +96,26 @@ class MainActivity : Activity() {
 
         reference.addValueEventListener(object : ValueEventListener {
             override fun onDataChange(dataSnapshot: DataSnapshot) {
-                val str = dataSnapshot.getValue(String::class.java) ?: return
-                val words = str.split(" ")
+                launch {
```

Comment:
This `launch` coroutine builder navigate us from normal world to coroutine world.
The listener method is a good point of changing world.

Reply:
Although I was adding `launch` coroutine just before sending packets, it looks good to add the builder to this listener indeed :)